inlong-common/src/test/java/org/apache/inlong/common/enums/TaskTypeEnumTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.common.enums;
+
+import org.junit.Test;
+
+import java.util.NoSuchElementException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class TaskTypeEnumTest {
+
+    @Test
+    public void testGetTaskType() {
+        TaskTypeEnum taskType = TaskTypeEnum.getTaskType(4);
+        assertEquals(TaskTypeEnum.KAFKA, taskType);
+
+    }
+
+    @Test
+    public void testGetTaskTypeWithOther() {
+        TaskTypeEnum taskType = null;
+        try {
+            // not exixts type:666
+            taskType = TaskTypeEnum.getTaskType(666);
+        } catch (NoSuchElementException e) {
+            System.out.println("=====error====");
```

Comment:
please use log.error() and assert the error is instanceof NoSuchElementException

Reply:
OK